requirements.txt

```diff
@@ -7,6 +7,6 @@ sqlparse==0.3.1
 whitenoise==5.2.0
 dj-database-url==0.5.0
 datadog==0.43.0
```

Comment:
@srikanthccv is this library needed anymore?

Reply:
No, it's not.

Reply:
Addressed in c6535349401a0808eb666f615cfb73be9049d3f0